### DISCUSSION TITLE
Consider renaming docker-compose.yaml to docker-compose.yml in default settings / examples

### DISCUSSION THREAD

**Discussion by w3cj** [coollabsio/coolify#1974] [Feature Requests]
Just debugged for an hour why my `docker-compose.yml` file was not loading for a new project and it was because Coolify defaults to `docker-compose.yaml` (different file extension). 🤦‍♂️ Definitely my own fault, but this was hard to spot.

Not sure if this is universal, but I've only ever used the `.yml` extension for compose files. The coolify project itself [uses `.yml`](https://github.com/coollabsio/coolify/blob/main/docker-compose.yml)

Looking into it a bit further, the docker documentation now encourages a file named `compose.yaml` instead of `docker-compose.yml` but the compose CLI will search for either to keep backwards compatibility.

Maybe coolify can try to do some auto detection of the compose file name? Otherwise, defaulting to `.yml` in placeholders and App defaults would be nice.

**Comment by kyoukisu** [comment]
no updates, no discussion? really weird
